*** ettercap-NG-0.7.3-orig/configure	2005-05-27 11:11:45.000000000 -0400
--- ettercap-NG-0.7.3/configure	2009-11-14 09:16:03.000000000 -0500
***************
*** 23457,23469 ****
    ac_status=$?
    echo "$as_me:$LINENO: \$? = $ac_status" >&5
    (exit $ac_status); } && test -s conftest${ac_exeext} 2>/dev/null; then
!     (./conftest; exit; ) 2>/dev/null
!     lt_status=$?
!     case x$lt_status in
!       x$lt_dlno_uscore) libltdl_cv_need_uscore=no ;;
!       x$lt_dlneed_uscore) libltdl_cv_need_uscore=yes ;;
!       x$lt_unknown|x*)  ;;
!     esac
    else :
      # compilation failed
  
--- 23457,23465 ----
    ac_status=$?
    echo "$as_me:$LINENO: \$? = $ac_status" >&5
    (exit $ac_status); } && test -s conftest${ac_exeext} 2>/dev/null; then
!     # Somehow the config gets this wrong so I'm hardcoding it.
!     # hopefully at somepoint in the future I can locat e the uderlying cause.
!     libltdl_cv_need_uscore=no 
    else :
      # compilation failed
  
***************
*** 28243,28249 ****
  
  fi
  
!    elif test "$OS" != "MACOSX" -a "$OS" != "WINDOWS"; then
        echo "$as_me:$LINENO: checking whether $CC accepts -pthread" >&5
  echo $ECHO_N "checking whether $CC accepts -pthread... $ECHO_C" >&6
        LDFLAGS_store="$LDFLAGS"
--- 28239,28245 ----
  
  fi
  
!    elif test "$OS" != "DARWIN" -a "$OS" != "WINDOWS"; then
        echo "$as_me:$LINENO: checking whether $CC accepts -pthread" >&5
  echo $ECHO_N "checking whether $CC accepts -pthread... $ECHO_C" >&6
        LDFLAGS_store="$LDFLAGS"
